charters/MetadataSchema/charter.md

```diff
@@ -0,0 +1,95 @@
+
+# Metadata Schema
+
+## Description
+
+The Human Cell Atlas (HCA) Metadata Schema are JSON format schema. These schema are designed to capture and provide structure for the descriptive scientific metadata associated with HCA datasets. These schemas aim to ensure [the FAIRness of the HCA data](https://www.nature.com/articles/sdata201618).
+
+These metadata schema aim to capture more than the minimum requirements for representing the samples and experiments conducted in the context of the Human Cell Atlas project but they do not aim to be truly comprehensive and capture every single variable that could be recorded about these samples and experiments. We are aiming for a middle ground that provides data consumers with rich enough information that they can use the data to answer biological questions but to not overburden the data contributors with an expectation to record every detail of every process they run in order to generate this data.
+
+## Definitions
+
+*Data contributor* - any user who uploads data into the DCP
+
+*Data consumer* - any user who accesses or downloads data from the DCP
+
+*Data wrangler* - an individual who works on behalf of the Human Cell Atlas (HCA) consortium to provide assistance to data contributors uploading data into the DCP. From a Metadata Schema perspective, data wranglers can themselves be considered as a specialised type of data contributor
+
+*Bundle* - the grouping of data and metadata in the Data Storage Service
+
+## Objectives
+
+1. Ensuring that HCA data are findable and the schemas are computationally queryable
+2. Ensuring that HCA metadata supports reproducibility and reusability
+3. Ensuring that HCA metadata has a consistent representation and can accurately represent the scientific description of a dataset in a computational readable manner
+4. Defining JSON schema which enable objectives 1 and 2
+5. Defining the semantic rules that the content follows
+6. Working with Ingestion Service to specify the entity structure for HCA metadata schema
+7. Working with the Data Store to receive new bundle structure requirements and defining a new specification to be implemented by the Ingestion Service team
+8. Defining style guides and the release process for the schema
+9. Ensuring the defined metadata schema provides a positive user experience for those who read and write them
+
+## In-scope
+
+### Schema Definition
+
+* Defining the specification for the schema and defining the schemas themselves 
+* Defining the semantic rules for metadata content of the schema
+* Ensuring the schema is compatible with open source software packages supporting the same draft of the JSON schema
+* Providing exemplar data in JSON format that validates against the schema 
+* Providing computationally readable information about deprecated and moved fields within the schema
+
+### Schema testing
```

Comment:
@malloryfreeberg @diekhans Can you review and potentially extend this section to reflect Mark's testing plan. Please do it in the next 48hrs so everyone can have enough time for a last review before community review is closed.